src/main/app/case/formatter/hearings-list.ts

```diff
@@ -0,0 +1,17 @@
+// import { Case /*, Checkbox */ } from '../case';
+// import { CaseData, UploadDocumentList /*, YesOrNo */ } from '../definition';
+
+// export const fromHearingApiApplicant1 = (data: Partial<CaseData>): Partial<Case> => {
+//   return {
+//     applicantUploadFiles:
+//       data.hearingCollection?.map(file => ({
+//         id: `${file.id}`,
+//         name: `${file.value}`,
+//       })) || [],
+//       hearingCollection: data.hearingCollection,
+//   };
+// };
+
+// export const getFilename = (document: Partial<UploadDocumentList> | undefined | null): string | undefined => {
+//   return document?.value?.citizenDocument.document_filename;
+// };
```

Comment:
what is this file is all about?

Reply:
will delete this file, currently not required